mico-core/src/main/java/io/github/ust/mico/core/MicoCoreApplication.java

```diff
@@ -79,29 +79,41 @@ public RestTemplate restTemplate(RestTemplateBuilder builder) {
         return builder.build();
     }
 
+    /**
+     * Runs when application is ready.
+     *
+     * @param event
+     */
     public void onApplicationEvent(final ApplicationReadyEvent event) {
+        addKafkaFaasConnectorToDatabase();
```

Comment:
This call should not happen in the test  environment. It still breaks the unit tests on travis.